src/main/java/ru/ayaz/UserMessageHandler.java

```diff
@@ -0,0 +1,144 @@
+package ru.ayaz;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.ArrayBlockingQueue;
+import java.util.concurrent.BlockingQueue;
+
+public class UserMessageHandler implements Runnable {
```

Comment:
&#x1F34E; **Issue**
Looks like this class has too many responsibilities on its hands.
You should split it into several more cohesive classes each with one responsibility only.
There is at least 3 responsibilities:
1) Working with queue in a separate thread - taking messages out of queue in an infinite loop.
2) Processing commands
3) Broadcasting a message
Third might be considered as a special case of the second, but nevertheless there is just too much of responsibilities in one place.